node.go

```diff
@@ -241,15 +242,17 @@ func (ns *nodeServer) NodeUnpublishVolume(ctx context.Context, req *csi.NodeUnpu
 }
 
 func (ns *nodeServer) runUpdater(volumeId string, state updaterState, restore bool) error {
-	log.Info().Str("volume_id", volumeId).Msg("starting updater")
+	log.Info().Str("volume_id", volumeId).Str("updateInterval ", state.UpdateInterval.String()).Msg("starting updater")
```

Comment:
An extra space after `updateInterval` in a log field name should be removed.

Reply:
`zerolog` has `Dur()` method to log `time.Duration` values.

Reply:
Thanks for noticing! Fixed this.